### PR TITLE
Support auto SSO

### PR DESCRIPTION
**Pull request by BigTows**
Hello! I am added new argument `data-auto-sso` for disable popup login block (auto redirect) if you are using only SSO method authentication.

Demo:

https://user-images.githubusercontent.com/9078504/115989833-ee995300-a5c8-11eb-9c53-a02fdd70fb2a.mp4

### CODE REVIEW

**Comment by BigTows** [comment]
I think argument probably useless, what do u think?

File: frontend/js/commento.js [hunk 1]
```diff
@@ -1935,22 +1937,27 @@
     });
 
     if (configuredOauths["sso"]) {
-      var button = create("button");
+      if (autoSSO && numOauthConfigured === 0 && configuredOauths["commento"] === false){
+        global.commentoAuth({"provider": "sso", "id": id});
+        setTimeout(global.loginBoxClose, 250);
+      } else {
```
Comment (r3ap3r2004) [inline frontend/js/commento.js:1943]: This and the end closing brace are all that is necessary.  If nothing else have been configured there isn't any point in the additional argument, just auto log them in.

**Review by r3ap3r2004 — COMMENTED** [review]
Thanks for this PR.  This solved an annoyance of mine of needing to have the users click another button when SSO was all I had enabled. 

Commenting on a really old PR isn't great I know, but I wanted you to know that I appreciate you taking the time to create it because it saved me time figuring it out on my own.

File: frontend/js/commento.js (hunk 1 above)
Comment (r3ap3r2004) [reply frontend/js/commento.js:1943]:
```go
      if (numOauthConfigured === 0 && configuredOauths["commento"] === false) {
        global.commentoAuth({ "provider": "sso", "id": id });
        setTimeout(global.loginBoxClose, 250);
      } else {
        ......
      }
```

**Comment by BigTows** [comment]
@r3ap3r2004 glad to hear :)

**Comment by souramoo** [comment]
Hey, sorry for the delay in getting back to you! This is a great idea; the current commits in this pull request have a lot of additional unneccessary changes so I have integrated the key lines into the latest commit at [aa0bf77](https://github.com/souramoo/commentoplusplus/commit/aa0bf779200caefe8d2b4eea76e71c6806cd46b4) :)

**Pull request closed without merging**